spec/webauthn/authenticator_attestation_response_spec.rb

```diff
@@ -148,6 +161,19 @@
       )
     end
 
+    before(:each) do
+      class PackedRootCertificatesStore
```

Comment:
No need to redefine the class (and load the file) every time before a test, these definitions can live outside of the before block.